src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -12643,10 +12643,8 @@ void WriteReportPeriodTimeConsumption(EnergyPlusData &state)
     WriteSubtitle(state, tableName);
     WriteTable(state, tableBody, rowHead, columnHead, columnWidth);
     if (state.dataSQLiteProcedures->sqlite) {
-        if (state.dataSQLiteProcedures->sqlite) {
-            state.dataSQLiteProcedures->sqlite->createSQLiteTabularDataRecords(
-                tableBody, rowHead, columnHead, "ReportingPeriodSummary", "Entire Facility", tableName);
-        }
+        state.dataSQLiteProcedures->sqlite->createSQLiteTabularDataRecords(
+            tableBody, rowHead, columnHead, "ReportingPeriodSummary", "Entire Facility", tableName);
     }
```

Comment:
Typical message for this type of change:

[src/EnergyPlus/OutputReportTabular.cc:12646]:(warning),[identicalInnerCondition],Identical inner 'if' condition is always true.